cli/packages/cmd/login.go

```diff
@@ -252,6 +274,51 @@ func init() {
 	rootCmd.AddCommand(loginCmd)
 }
 
+func askForDomain() error {
+	//query user to choose between Infisical cloud or self hosting
+	options := []string{"Infisical Cloud", "Self Hosting"}
+	optionsPrompt := promptui.Select{
+		Label: "Select your hosting option",
+		Items: options,
+		Size:  2,
+	}
+
+	idx, _, err := optionsPrompt.Run()
+	if err != nil {
+		return err
+	}
+
+	if idx == 0 {
+		//cloud option
+		config.INFISICAL_URL = util.INFISICAL_DEFAULT_API_URL
+		return nil
+	}
+
+	urlValidation := func(input string) error {
+		_, err := url.ParseRequestURI(input)
+		if err != nil {
+			return errors.New("this is an invalid url")
+		}
+		return nil
+	}
+
+	//else run prompt to enter domain
+	domainPrompt := promptui.Prompt{
+		Label:    "Domain",
```

Comment:
I wonder if domain is clear, when i think of domain i think of google.com not the https or http before it. Do you know if there is a way to provide a placeholder text with an example so users know?

Reply:
Yeah we could provide the `default` value in the prompt object. Example: `Default: 'Example: https://my-domain-example.com`

Reply:
or maybe `https://my-self-hosted-instance.com`? Feel free to go with what you think is best for developers to understand